tests/utils.go

```diff
@@ -2910,14 +2912,20 @@ func configureIPv6OnVMI(vmi *v1.VirtualMachineInstance, expecter expect.Expecter
 		&expect.BExp{R: prompt},
 		&expect.BSnd{S: "echo $?\n"},
 		&expect.BExp{R: "0"},
-		&expect.BSnd{S: fmt.Sprintf(`echo "nameserver %s" >> /etc/resolv.conf\n`, dnsServerIP)},
```

Comment:
it fails always, just sometimes u do see it fails , its matter of race, i simulated it locally by adding a sanity check at the end , or by adding a seperated snippet for sanity (something like you want in the comment above) and saw it fails 10/10 times

Reply:
Race ? Race between what ? 

On a separate note... I'm getting the impression that if we delete everything DNS related from this function we'll achieve the same result - since DNS is only required for the guestagent tests, where DNS is configured via cloud-init.

Instead of fixing, I would actually remove this from here - since it's not needed.

Reply:
race between the expect of this own block or the following ones
see please the 2 reports in the commit message

indeed we might be able to remove it, i wanted to remove it for long time, (mentioned on couple PRs), but it need to be done safe if we want.
atm prow is broken, should we just remove it then and see that nothing fails, or we do want to have a function in case we need it in the future, and learn what i can from it ?

Reply:
> race between the expect of this own block (see 3184) or the following block (see for example 3187)
> see please the 2 reports in the commit message

I don't understand what you're saying; please include links to kubevirt code instead of making me fish for (what I think are) lines of code in what I assume to be the same file. This is not needed since it appears we have an agreement on the course of action - remove DNS configuration via console).

> 
> indeed we might be able to remove it, i wanted to remove it for long time, (mentioned on couple PRs), but it need to be done safe if we want.
> atm prow is broken, should we just remove it then and see that nothing fails, or we do want to have a function in case we need it in the future, and learn what we can from it ?
> green light to remove ?

+1 to remove it.